system-addon/test/unit/lib/TelemetryFeed.test.js

```diff
@@ -531,13 +537,20 @@ describe("TelemetryFeed", () => {
       instance.onAction({type: at.UNINIT});
       assert.calledOnce(stub);
     });
+    it("should call .handleNewTabInit on a NEW_TAB_INIT action", () => {
+      sandbox.spy(instance, "handleNewTabInit");
+      instance.onAction(ac.SendToMain({
+        type: at.NEW_TAB_INIT,
+        data: {url: "about:newtab", browser}
+      }));
+      assert.calledOnce(instance.handleNewTabInit);
+    });
     it("should call .addSession() on a NEW_TAB_INIT action", () => {
-      const stub = sandbox.stub(instance, "addSession");
+      const stub = sandbox.stub(instance, "addSession").returns({perf: {}});
       sandbox.stub(instance, "setLoadTriggerInfo");
-
```

Comment:
I'd prefer leaving the whitespace for the readability reason suggested elsewhere.